cylc/flow/simulation.py

```diff
@@ -157,7 +210,8 @@ def parse_fail_cycle_points(
 
 
 def sim_time_check(
-    message_queue: 'Queue[TaskMsg]', itasks: 'List[TaskProxy]'
+    message_queue: 'Queue[TaskMsg]', itasks: 'List[TaskProxy]',
+    broadcast_mgr,
```

Comment:
Sorry, mixed horizontal/vertical arrangement drives me nuts.

```suggestion
    message_queue: 'Queue[TaskMsg]',
    itasks: 'List[TaskProxy]',
    broadcast_mgr,
```

Reply:
Yes. I'd not have done that deliberately.